glean_parser/templates/javascript_server.jinja2

```diff
@@ -0,0 +1,154 @@
+{# The final Javascript/Typescript code is autogenerated, but this
+Jinja2 template is not. Please file bugs! #}
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// AUTOGENERATED BY glean_parser v{{ parser_version }}. DO NOT EDIT. DO NOT COMMIT.
+
+// This requires `uuid` and `mozlog` libraries to be in the environment
+{% if lang == "typescript" %}
+// @types/uuid and mozlog types definitions are required in devDependencies
+// for the latter see https://github.com/mozilla/fxa/blob/85bda71cda376c417b8c850ba82aa14252208c3c/types/mozlog/index.d.ts
+{% endif %}
+import { v4 as uuidv4 } from 'uuid';
+import mozlog{% if lang == "typescript" %}, { Logger }{% endif %} from 'mozlog';
+
+const GLEAN_EVENT_MOZLOG_TYPE = 'glean-server-event';
+
+let _logger{% if lang == "typescript" %}: Logger{% endif %};
```

Comment:
The first call to any factory method will set this and thus prescribe which `logger_options` are used.
If there's multiple pings and a user calls the factory method with different `logger_options` the order of calls influences what options are set.

I assume in the current usage this wouldn't be an issue because (a) there's only a single ping, (b) even if it were multiple pings it would be the same logger options, is that correct?

Reply:
Yes, I'm aware this approach is not ideal. At the moment we don't expect to have multiple pings, but even in such case logger options should remain constant.